views_migration_17/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Views Migration to v17",
+    "version": "17.0.1.0.0",
+    "author": "ODOO SA,ADHOC SA,Odoo Community Association (OCA)",
+    "description": """
+Patch modules views related to this change https://github.com/odoo/odoo/pull/104741
+The script is taken from this comment (https://github.com/odoo/odoo/pull/104741#issuecomment-1794616832) on same PR
+To run it:
+1. Add module as server wide module.
+2. Run odoo server installing or upgrading target module.
+
+For eg: odoo -i upgrade_analysis -d upgrade_analysis --load=base,web,views_migration_17
+""",
+    "website": "https://github.com/OCA/server-tools",
+    "license": "AGPL-3",
+    "depends": [
+        "base",
+    ],
+    "data": [],
+    "installable": False,
```

Comment:
This module had installable set to False, why?

Reply:
This is a server module, the only use is to patch the odoo to migrate views to v17 (https://github.com/odoo/odoo/pull/104741). In the README it is explained how to use it.